src/components/book-form/book-form.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { addBook } from '../../redux/actions/library';
+import './book-form.styles.scss';
+import { CATEGORIES } from '../../constants/constants';
+
+class BookForm extends React.Component {
+	constructor() {
+		super();
+		this.state = {
+			title: '',
+			category: 'Action'
+		}
+	}
+
+	handleChange = event => {
+		const {name, value } = event.target;
+		this.setState({ [name]: value });
+	}
+
+	handleSubmit = event => {
+		event.preventDefault();
+		const { title, category } = this.state; 
+		fetch('http://localhost:3000/api/v1/books', {
```

Comment:
Again, a bad place to put our data fetching. And this should be an action that affects the redux store. So essentially again we can avoid such problems for now by just doing Milestones 1 to 5. After getting a thumbs up for those then we can talk about working on Milestone 6. :)